Tests/SwinjectStoryboard/ViewController+SwinjectSpec.swift

```diff
@@ -1,20 +1,29 @@
 //
-//  UIViewController+SwinjectSpec.swift
+//  ViewController+SwinjectSpec.swift
 //  Swinject
 //
 //  Created by Yoichi Tagaya on 8/1/15.
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
 
+
 import Quick
 import Nimble
 @testable import Swinject
+    
+#if os(iOS) || os(tvOS)
+private let createViewController = { UIViewController(nibName: nil, bundle: nil) }
+#elseif os(OSX)
+private let createViewController = { NSViewController(nibName: nil, bundle: nil)! }
+#endif
+
+#if os(iOS) || os(OSX) || os(tvOS)
 
-class UIViewController_SwinjectSpec: QuickSpec {
+class ViewController_SwinjectSpec: QuickSpec {
```

Comment:
Type Name Violation: Type name should only contain alphanumeric characters: 'ViewController_SwinjectSpec' (type_name)